conf/defaults.ini

```diff
@@ -209,6 +209,12 @@ x_content_type_options = true
 # when they detect reflected cross-site scripting (XSS) attacks.
 x_xss_protection = true
 
+# Set Content Security Policy string to add the Content-Security-Policy header for your requests.
+# CSP allows to control resources the user agent is allowed to load and help to prevent XSS atacks.
+# Add $nonce to script-src to add random nonce.
```

Comment:
Should it be $NONCE instead? That's how it's written in the actual string below.